adafruit_ads1x15/ads1x15.py

```diff
@@ -198,23 +190,35 @@ def comparator_high_threshold(self) -> int:
 
     @comparator_low_threshold.setter
     def comparator_low_threshold(self, value: int) -> None:
-        """Set comparator low threshold value for ADS1015 ADC
+        """Set comparator low threshold value for ADS1x15 ADC
 
         :param int value: 16-bit signed integer to write to register
         """
-        if value < 0 or value > 65535:
-            raise ValueError("Comparator Threshold value must be between 0 and 65535")
+        if value < -32768 or value > 32767:
+            raise ValueError(
+                "Comparator Threshold value must be between -32768 and 32767"
+            )
+
+        if (self.bits == 12) & (value & 0x000F > 0):
+            print("4 LSBs will be truncated for ADS1015 for 12-bit value")
+
```

Comment:
I don't think I'd include this. If you want to, then make it an exception. `print`s in libraries are just annoying.
```suggestion
```

Reply:
These were meant to be informational, not worthy of exception. I removed them.